environments/databricks-department-clusters-pat/README.md

```diff
@@ -180,3 +182,13 @@ In order to create a new Github Actions workflow, follow these steps:
 * Copy the [github-actions.yml](github-actions.yml) into the ``terraform-databricks-demo.yml`` file.
 * Commit and push your changes to the `main` branch. This will automatically create a new Github Actions workflow. 
 
+
+### Configuring the Github Actions release job
+
+The release job requires a manual approval before deploying the changes. Once the reviewers approve the request, the release job will start deploying the changes.
+In order to configure the manual approval, follow these steps:
+
+* On GitHub.com, navigate to the main page of the repository and click "Settings".
+* In the left sidebar, click Environments and click "New environment".
+* Name the environment `production`. This name is used in the [github-actions.yml](github-actions.yml) file.
+* Specify people or teams that must approve release job.
```

Comment:
should they fill secrets for environment as well? Or it will use just "normal" secrets

Reply:
Just "normal" secrets for the workflow. 
No environment specific secrets are needed.